src/unittests/sr5.AttackTests.spec.ts

```diff
@@ -167,67 +170,90 @@ export const shadowrunAttackTesting = (context: QuenchBatchContext) => {
             testScene = new SR5TestingDocuments(Scene);
         });
 
+        const getCharacterWithArmor = async (armorValue: number, {
+            hardened = false
+        }: {
+            hardened?: boolean
+        } = {}): Promise<SR5Actor> => {
+            const characterActor = await testActor.create({
```

Comment:
You are missing the tearDown for the actors. In https://github.com/SR5-FoundryVTT/SR5-FoundryVTT/pull/919 I fixed it for existing tests. Can you add it to your new tests?

Reply:
Whoops, good catch! Fixed.